docs/build/apps/dapp-frontend.mdx

```diff
@@ -56,7 +58,11 @@ This project has the following directory structure, which we'll go over in more
 └── tsconfig.json
 ```
 
-The `contracts` are the same ones you walked through in the previous steps of the tutorial.
+The `contracts` are the same ones you walked through in the previous steps of the tutorial. Since we already deployed these contracts with aliases, we can reuse the generated contract ID files by copying them from the `soroban-hello-world/.stellar` directory into this project:
+
+```sh
+cp -R ../soroban-hello-world/.stellar/ .stellar
```

Comment:
TIL the difference between `cp -R` and (what I've been using for years) `cp -r` (at least for macOS). Thanks!